### PR TITLE
28 build offline notification

### LINKED ISSUE #28: Build offline notification

### PR DESCRIPTION
**Pull request by Stef-ko**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ android {
 
     defaultConfig {
         applicationId "app.mywatchlist"
-        minSdk 29
+        minSdk 21
```
Comment (steffenkleinle) [inline app/build.gradle:15]: Is this intended?

**Review by steffenkleinle — APPROVED** [review]

**Comment by steffenkleinle** [comment]
Maybe we should display the error full screen or handle unavailable data. Even if offline it is still possible with results in endless loading (search) or displaying of null values (detail page).

**Comment by Stef-ko** [comment]
Kinda goes against the requirement to show cached data when offline. Also what do you mean by the second sentence?

**Comment by steffenkleinle** [comment]
If you are offline and search for something with a new query, there is a endless loading spinner. If you are in the list view and offline and click on a watchable, it opens the detail screen with null values.

File: app/build.gradle (hunk 1 above)
Comment (steffenkleinle) [reply app/build.gradle:15]: Doesnt matter, well keep it

Comment (steffenkleinle) [reply app/build.gradle:15]: Doesnt matter, well keep it

**Pull request merged**